ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ReadWriteServiceCallAuthTests.scala

```diff
@@ -16,63 +17,107 @@ trait ReadWriteServiceCallAuthTests extends ServiceCallWithMainActorAuthTests {
+    .setAttack(
+      attack(threat = "Exploit a non-expired 'unknown_user' user token")
+    ) in {
     expectPermissionDenied(serviceCallWithToken(canReadAsUnknownUserStandardJWT))
   }
-  it should "deny calls with explicitly non-expired read-only token" in {
+  it should "deny calls with explicitly non-expired read-only token" taggedAs securityAsset
+    .setAttack(
+      attack(threat = "Exploit a explicitly non-expired read-only token")
+    ) in {
     expectPermissionDenied(serviceCallWithToken(canReadAsMainActorExpiresTomorrow))
   }
-  it should "deny calls with read-only token without expiration" in {
+  it should "deny calls with read-only token without expiration" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit a read-only token without expiration")
+  ) in {
     expectPermissionDenied(serviceCallWithToken(canReadAsMainActor))
   }
-
-  it should "allow calls with the correct ledger ID" in {
+  it should "allow calls with the correct ledger ID" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can connect with "
+    ) in {
     expectSuccess(serviceCallWithToken(canActAsMainActorActualLedgerId))
   }
-  it should "deny calls with a random ledger ID" in {
+  it should "deny calls with a random ledger ID" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit a random ledger ID")
+  ) in {
     expectPermissionDenied(serviceCallWithToken(canActAsMainActorRandomLedgerId))
   }
-  it should "allow calls with the correct participant ID" in {
+  it should "allow calls with the correct participant ID" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can connect with the correct participant ID"
+    ) in {
     expectSuccess(serviceCallWithToken(canActAsMainActorActualParticipantId))
   }
-  it should "deny calls with a random participant ID" in {
+  it should "deny calls with a random participant ID" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit a")
+  ) in {
     expectPermissionDenied(serviceCallWithToken(canActAsMainActorRandomParticipantId))
   }
-  it should "allow calls with the correct application ID" in {
+  it should "allow calls with the correct application ID" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can connect with the correct application ID"
+    ) in {
     expectSuccess(serviceCallWithToken(canActAsMainActorActualApplicationId))
   }
-  it should "deny calls with a random application ID" in {
+  it should "deny calls with a random application ID" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit a random application ID")
+  ) in {
     expectPermissionDenied(serviceCallWithToken(canActAsMainActorRandomApplicationId))
   }
-  it should "allow calls with an empty application ID for a token with an application id" in {
+  it should "allow calls with an empty application ID for a token with an application id" taggedAs securityAsset
```

Comment:
Same old problem can be found here as was the case in the [CompletionStreamAuthIT.scala](https://github.com/digital-asset/daml/pull/13445/files#diff-4b91578158d26be2da1aad87e8140d429bb48dfb8150d6f18b9c89e0236b17af)